collections/pm4ml/fx_golden_path/TESTCASES.md

```diff
@@ -0,0 +1,15 @@
+* Negative-scenarios
```

Comment:
I am not sure how TTK behaves with the md files if we load this folder in collection manager. Its better to not have it here.